src/middleware/express/make-http-request.ts

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { HttpRequest } from '../../http-request';
+// Types-only import.
+import * as express from 'express';
```

Comment:
It would be nice to have this generic so folks can call a function that gives them formatting help with any core compatible request/response objects.

If so, we could use http.IncomingMessage and http.ServerResponse instead of express types and maybe this could be an export of src/http-request.ts.

its also really nice to just see it land here. how the function is exposed can be another pr later

Reply:
I'm +1 on a generic version, but this will probably have to stay. Not all information that we need for `HttpRequest` may be extractable from a generic `http.IncomingMessage` etc. It makes sense to tackle in a follow-on.